docs/lang/articles/reference.md

```diff
@@ -188,8 +188,29 @@ Literals are evaluated to Python values at compile time.
 
 #### Identity comparisons
 
+The operators `is` and `is not` test for an object’s identity:
+`x is y` is `true` if and only if `x` and `y` are the same object.
+An Object’s identity is determined using the [id()](https://docs.python.org/3/library/functions.html#id) function.
+`x is not y` yields the inverse truth value.
+
+:::note
+Operator `is` or `is not` in Taichi scope is deprecated. Please avoid using it.
```

Comment:
We don't have to document deprecated operators. What about simply removing this subsection?

Reply:
remove